bundles/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/NetworkUtils.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.io.hueemulation.internal;
+
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.ResponseBuilder;
+import javax.ws.rs.core.UriInfo;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.io.hueemulation.internal.dto.response.HueResponse;
+import org.openhab.io.hueemulation.internal.dto.response.HueResponse.HueErrorMessage;
+import org.openhab.io.hueemulation.internal.dto.response.HueResponseSuccessSimple;
+import org.openhab.io.hueemulation.internal.dto.response.HueSuccessGeneric;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+/**
+ * Network utility methods
+ *
+ * @author David Graeff - Initial contribution
+ */
+@NonNullByDefault
+public class NetworkUtils {
+    /**
+     * Try to get the ethernet interface MAC for the network interface that belongs to the given IP address.
+     * Returns a default MAC on any failure.
+     *
+     * @param address IP address
+     * @return A MAC of the form "00:00:88:00:bb:ee"
+     */
+    static String getMAC(InetAddress address) {
```

Comment:
Shouldn't this method be moves to the core netutil-class? I would merge here to go on, but could you prepare PR to move it?

Reply:
I have 5 open PRs in Core and 10 Issues :D But yeah I can add that to the list.